controller/volume_controller.go

```diff
@@ -1902,8 +1903,21 @@ func (vc *VolumeController) ResolveRefAndEnqueue(namespace string, ref *metav1.O
 	vc.enqueueVolume(volume)
 }
 
+func (vc *VolumeController) getTaintToleration() ([]v1.Toleration, error) {
```

Comment:
Move this function into the datastore, like we did for the `GetSettingImagePullPolicy` function.
There is also a duplicate of this in the instance-manager-controller, engine-image-controller instead make them use the datastore function that you added.

Reply:
Great idea! Thank you! Done